docs/docs/build/relay.md

```diff
@@ -22,5 +22,52 @@ There was only one issue highlighted in the Least Authority audit related to the
 
 ## Using the relay
 
-- To check the inclusion of a specific transaction, the `BitcoinTx.validateProof` function can be used. See `test/LightRelay.t.sol` for an example.   
-- `BitcoinTx.getTxOutputValue` can be be used to check the amount transfered to a specific address in a given `txOut`. See `test/BitcoinTx.t.sol` for an example.
+The code for the light relay is in `src/relay/LightRelay.sol` which stores the difficulty for the current and previous epoch. To update this it is possible to use `retarget(headers)` with `proofLength * 2` block headers from Bitcoin (before and after the retarget) serialized sequentially. 
+
+:::tip BOB SDK
+
+Use the `getBitcoinHeaders` function to automatically read these from the configured Electrs REST API.
```

Comment:
This is unclear to me. What I think you are trying to say is that you can use the SDK to get the required input data for the contract. This needs to be made clear in the sense of: this is the Solidity function of the relay and this is how you get the input data using the BOB SDK which in this case is calling electrs in the background.